README.md

```diff
@@ -1,10 +1,15 @@
 # MinIO Quickstart Guide
 [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Go Report Card](https://goreportcard.com/badge/minio/minio)](https://goreportcard.com/report/minio/minio) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/)
 
-MinIO is an object storage server released under Apache License v2.0. It is compatible with Amazon S3 cloud storage service. It is best suited for storing unstructured data such as photos, videos, log files, backups and container / VM images. Size of an object can range from a few KBs to a maximum of 5TB.
+MinIO is an object storage server released under Apache License v2.0. It is compatible[1] with Amazon S3 cloud storage service. It is best suited for storing unstructured data such as photos, videos, log files, backups and container / VM images. Size of an object can range from a few KBs to a maximum of 5TB.
 
 MinIO server is light enough to be bundled with the application stack, similar to NodeJS, Redis and MySQL.
 
+[1]: Some software such as S3QL will not work with MinIO in its default mode which is faster but not fully compatible with S3. For full compatibility with Amazon S3 protocols, it must be started in server (not gateway) mode and given the --compat option.
```

Comment:
```suggestion
[1]: Some software such as S3QL will not work with MinIO in its default mode which is faster but not fully compatible with S3. For full compatibility with Amazon S3 protocols, start MinIO with `--compat` option.
```

Reply:
Being new to MinIO (appx 1 day), I need some clarification. My original change was based on reading of past discussions / bug reports where it was stated (as I recall) that gateway mode was by design not fully S3 compatible. I don't know when the --compat option was added, but your change suggestion seems to imply that both server and gateway mode are compatible when --compat is selected. If so then I would agree with your suggested change.

Reply:
Yes both gateway and server honor `--compat` option